packages/story-editor/src/components/floatingMenu/elements/toggleBold.js

```diff
@@ -19,18 +19,47 @@
  */
 import { Icons } from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
+import { trackEvent } from '@googleforcreators/tracking';
+import { useCallback } from '@googleforcreators/react';
 
 /**
  * Internal dependencies
  */
-import { IconButton } from './shared';
+import useRichTextFormatting from '../../panels/design/textStyle/useRichTextFormatting';
+import updateProperties from '../../inspector/design/updateProperties';
+import { useStory } from '../../../app';
+import { IconButton, useProperties } from './shared';
 
 function ToggleBold() {
+  const { content } = useProperties(['content']);
+  const updateSelectedElements = useStory(
+    (state) => state.actions.updateSelectedElements
+  );
+
+  const pushUpdate = useCallback(
+    (update) => {
+      trackEvent('floating_menu', {
+        name: 'set_bold',
+        element: 'text',
+      });
+      updateSelectedElements({
+        properties: (element) => updateProperties(element, update, true),
+      });
+    },
+    [updateSelectedElements]
+  );
+  const {
+    textInfo: { isBold },
+    handlers: { handleClickBold },
+  } = useRichTextFormatting([{ content, type: 'text' }], pushUpdate);
+
   return (
     <IconButton
+      isToggled={isBold}
       Icon={Icons.LetterBBold}
-      title={__('Toggle bold text', 'web-stories')}
-      onClick={() => {}}
+      title={__('Toggle bold', 'web-stories')}
```

Comment:
Hmm, why doesn't the tooltip show up on keyboard focus? I'm pretty sure it did earlier, but it seems to be a problem with all the buttons using this `IconButton` component now (which the eyedropper doesn't use). 

| Without tooltip | With tooltip |
|-|-|
| ![Screen Shot 2022-03-10 at 16 50 40](https://user-images.githubusercontent.com/637548/157760827-423c318b-d0b2-46f1-a8bb-fc3b339c33ce.png) | ![Screen Shot 2022-03-10 at 16 50 32](https://user-images.githubusercontent.com/637548/157760835-0ce03892-7bd7-42aa-8acb-2b76664f1b77.png) |

Reply:
Hm, not sure -- added an issue: https://github.com/GoogleForCreators/web-stories-wp/issues/10884